operate/ledger/profiles.py

```diff
@@ -91,7 +91,7 @@
         "optimus_alpha": "0x88996bbdE7f982D93214881756840cE2c77C4992",
     },
     Chain.ETHEREUM: {},
-    Chain.BASE: {"meme_base_alpha": "0x06702A05312091013FdB50C8b60B98ca30762931"},
+    Chain.BASE: {"meme_base_alpha_2": "0xc653622FD75026a020995a1d8c8651316cBBc4dA"},
```

Comment:
@jmoreira-valory not sure if the name is shared between FE and BE only or it should be updated in the service/agent/something else as well?

Reply:
@Tanya-atatakai it is shared between both, https://github.com/valory-xyz/olas-operate-app/blob/87eea7dfccedcf0c1e01ab3d83c2621e611056c7/frontend/service/Services.ts#L78

Reply:
@Tanya-atatakai The staking contract ID (e.g., meme_base_alpha_2) is passed to the backend only in the service template when creating/updating a service. Based on that value, the backend will fill a number of env variables ("COMPUTED") that deppend on the particular staking contract.

Please see comment above from @truemiller regarding the frontend.